frontend/src/integrations/groupsio/components/groupsio-connect-drawer.vue

```diff
@@ -0,0 +1,379 @@
+<template>
+  <app-drawer
+    v-model="isVisible"
+    custom-class="integration-reddit-drawer"
+    title="groups.io"
+    size="600px"
+    pre-title="Integration"
+    :show-footer="true"
+    has-border
+    @close="handleCancel()"
+  >
+    <template #beforeTitle>
+      <img class="w-6 h-6 mr-2" :src="logoUrl" alt="Groups.io logo" />
+    </template>
+    <template #content>
+      <div class="w-full flex flex-col mb-6">
+        <p class="text-[16px] font-semibold">
+          Authentication
+        </p>
+        <div class="text-2xs text-gray-500 leading-normal mb-1">
+          Connect a Groups.io account. You must be a group owner to authenticate.
+        </div>
+      </div>
+      <el-form label-position="top" class="form">
+        <app-form-item
+          v-if="!isAPIConnectionValid"
+          class="
+          mb-6"
+          :validation="$v.email"
+          label="Email"
+          :required="true"
+          :error-messages="{
+            required: 'This field is required',
+            url: 'Enter valid email',
+          }"
+        >
+          <!-- <div class="text-2xs text-gray-500 leading-normal mb-1">
+            Hostname of your community instance in Discourse.
+          </div> -->
+          <el-input ref="focus" v-model="form.email" type="email" @blur="onBlurEmail()" />
+        </app-form-item>
+        <app-form-item
+          v-if="!isAPIConnectionValid"
+          class="mb-6"
+          :validation="$v.apiKey"
+          label="Password"
+          :required="true"
+          :error-messages="{
+            required: 'This field is required',
+          }"
+        >
+          <el-input ref="focus" v-model="form.password" @blur="onBlurPassword()">
+            <template #suffix>
+              <div
+                v-if="isValidating"
+                v-loading="isValidating"
+                class="flex items-center justify-center w-6 h-6"
+              />
+            </template>
+          </el-input>
+        </app-form-item>
+        <app-form-item
+          v-if="!isAPIConnectionValid"
+          class="mb-6"
+          :validation="$v.twoFactorCode"
+          label="2FA Code (optional)"
+        >
+          <el-input ref="focus" v-model="form.twoFactorCode" @blur="onBlurTwoFactorCode()">
+            <template #suffix>
+              <div
+                v-if="isValidating"
+                v-loading="isValidating"
+                class="flex items-center justify-center w-6 h-6"
+              />
+            </template>
+          </el-input>
+        </app-form-item>
+
+        <div class="flex flex-row gap-2">
+          <el-button
+            v-if="!isAPIConnectionValid"
+            class="btn"
+            :disabled="!isVerificationEnabled"
+            :loading="isVerifyingAccount"
+            @click="validateAccount()"
+          >
+            Verify Account
+          </el-button>
```

Comment:
You're missing the right button classes for this one
Maybe add it as a secondary button
<img width="190" alt="Screenshot 2023-09-01 at 17 09 06" src="https://github.com/CrowdDotDev/crowd.dev/assets/20134207/c24995d7-659c-497f-8ae1-f2d78553108c">

Reply:
Added a secondary button